subprojects/docs/src/docs/userguide/reference/directory_layout.adoc

```diff
@@ -68,7 +68,7 @@ If using `--no-daemon`, it runs in the foreground after the build session with a
 
 The following cleanup strategies are applied periodically (by default, once every 24 hours):
 
-- Version-specific caches in `caches/<gradle-version>/` are checked for whether they are still in use.  If not, directories for release versions are deleted after 30 days of inactivity, snapshot versions after 7 days of inactivity.
+- Version-specific caches in `caches/{gradleVersion}/` (or `caches/<gradle-version>`)) are checked for whether they are still in use.  If not, directories for release versions are deleted after 30 days of inactivity, snapshot versions after 7 days of inactivity.
```

Comment:
❓ What is the need for the 'or' here, aren't these 2 ways of saying the same dir?  There's an extra closing paren - is this a mistake?

Reply:
Removed in the final version.